index.js

```diff
@@ -67,14 +67,16 @@ const libraryAddress = argv.libraryAddress ? argv.libraryAddress : "0x0000000000
     priceFeedIdentifier: padRight(utf8ToHex(argv.priceFeedIdentifier.toString()), 64), // Price identifier to use.
     syntheticName: argv.syntheticName, // Long name.
     syntheticSymbol: argv.syntheticSymbol, // Short name.
-    collateralRequirement: { rawValue: toWei("1.25") }, // 125% collateral req.
+    
+    collateralRequirement: { rawValue: toWei("1") }, // 100% collateral req is possible because position is always backed by 1 unit of colateral, as 
```

Comment:
This is probably the most annoying part. I didn't realize this, but there's a strict inequality on this that requires it to be >100%. So the best we can do is make it `1 + 1e-18`.
```suggestion
    collateralRequirement: { rawValue: web3.utils.toBN(toWei("1")).addn(1).toString() }, // 100% collateral req is possible because position is always backed by 1 unit of colateral, as 
```